sycl/source/detail/scheduler/scheduler.cpp

```diff
@@ -9,6 +9,10 @@
 #include "detail/sycl_mem_obj_i.hpp"
 #include <detail/global_handler.hpp>
 #include <detail/graph_impl.hpp>
+#include <sycl/feature_test.hpp>
+#if SYCL_EXT_CODEPLAY_KERNEL_FUSION
```

Comment:
We could rename this define, to also reflect the new usage.

Reply:
Lots of renaming to be done, yes

Reply:
Yeah, makes sense to do the refactoring in a separate follow-up PR.

Reply:
Ticket created, will arrive in a follow up PR.